index.html

```diff
@@ -156,7 +167,7 @@ <h3 class="card-title">Tonic</h3>
                         </div>
                     </div>
                 </div>
-                <div class="cards">
+                <div class="cards card-4">
                     <div>
                         <img src="asset/snapshot4.svg" alt="screenshot" class="screenshot">
                     </div>
```

Comment:
- [ ] You did a great job with the design and responsiveness. :+1: Let's work on some more details to match the design. Kindly make the title `About Myself` a single line instead of two. 😎️


| Your design                                                             | Figma design                                         |
| --------------------------------------------------------------------------------| --------------------------------------------------------- |
|<img src="https://user-images.githubusercontent.com/17243325/236253401-d495c049-c61a-414a-80c0-e4db5dd7d9d6.png" width="300">|<img src="https://user-images.githubusercontent.com/17243325/236253392-459b69e3-ae70-4efd-a9f7-a6991cdb78c1.png" width="300">|